pkg/identity/identity_test.go

```diff
@@ -54,7 +56,29 @@ var _ = Describe("GetCredentials", func() {
 	AfterEach(func() {
 		Expect(k8sclient.Delete(ctx, ns)).To(Succeed())
 	})
+	fmt.Println("with checking if cluster spec identity Ref secret")
```

Comment:
Do we need these `Println` statements or added by mistake?

Reply:
I have added them for more verbosity. Without this, searchability of the tests in terminal was technically impossible.

Reply:
There are other ways of reporting the tests that were run. Highly suggest that having print statements in tests for reporting running isn't the best way to achieve it. Do consider removing this

Reply:
Sure! I will ponder a little more into it.